nrt/data/IEEE14/IEEE14_WithAutomata/description/IEEE14_WithAutomata.tex

```diff
@@ -106,7 +109,249 @@ \section{Results}
+
+The phase-shifter parameters are:
+\begin{center}
+\begin{tabular}{l|l|l}
+   $P_{target}=0.44p.u$ & $t_{1st}=10s$ & $tap_{Max}=13$ \\
+   $P_{DeadBand}=0.005p.u$  & $t_{Next}=5s$ & $tap_{Min}=1$ \\
+\end{tabular}
+\end{center}
+
+\subsection{Scenario}
+At $t=20s$, the following line disconnection is simulated:
+\begin{itemize}
+\item{the line connecting bus 9 to bus 10 is opened at its bus 10 end}
+\end{itemize}
+
+\subsection{Solver}
+The solver used is the variable time step solver IDA with the following parameters:
+\begin{center}
+\begin{tabular}{l|l|l}
+   $Order$=2 & $Accuracy_{Rel}$=10e-4 & $Accuracy_{Abs}$=10e-4 \\
+\end{tabular}
+\end{center}
+
+\newpage
+\subsection{Results}
+
+At t=20s, the line is disconnected. Following the topology modification, the current through the transformer monitored by the phase shifter goes out of the tolerated range around $P_{Target}$.  10 seconds later, the transformer tap is changed which results in a step decrease of transiting active power. Though, it is still not in the tolerated range so the phase-shifter increases the tap again, this time only waiting for 5 seconds. Consequently, the active power transit drops and transiently falls back into the dead-band before going out of it again. Therefore, the phase-shifter P first temporarily stops taking steps, and then starts again waiting for a 10s delay before shifting to the next tap. This process repeats two times more and the automaton stops acting once the active power transit finally stabilizes in the range around $P_{Target}$. \\
+
+\begin{figure}[H]
+\subfigure[Transformer's current (p.u.)]
+{%
+  \begin{tikzpicture}
+    \begin{axis}[height = 2in]
+        \addplot[dashed, color=red!50]
+        table[x=time, y=PhaseShifter_phaseShifter_valueMax]
+        {../IEEE14_PhaseShifterP/reference/outputs/curves/curves.csv};
+        \addplot[color=blue!50]
+        table[x=time, y=PhaseShifter_phaseShifter_PMonitored]
+        {../IEEE14_PhaseShifterP/reference/outputs/curves/curves.csv};
+        \addplot[dashed, color=green!50]
+        table[x=time, y=PhaseShifter_phaseShifter_valueMin]
+        {../IEEE14_PhaseShifterP/reference/outputs/curves/curves.csv};
+        \legend{$P_{Max_{Pu}}$, $P_{Pu}$, $P_{Min_{Pu}}$}
+    \end{axis}
+  \end{tikzpicture}
+}
+\subfigure[Transformer's tap (p.u.)]
+{%
+\begin{tikzpicture}
+    \begin{axis}[height = 2in]
+        \addplot[color=blue!50]
+        table[x=time,y=NETWORK__BUS____5-BUS____6-1_PS_step]
+        {../IEEE14_PhaseShifterP/reference/outputs/curves/curves.csv};
+        \legend{$Tap$}
+    \end{axis}
+  \end{tikzpicture}
+}
+\caption{Behavior of the phase-shifter P}
+\end{figure}
+
+
+\newpage
+\section{Current limit automaton}
+\label{CurrentLimitAutomaton}
+
+Current limit automata monitor lines' transiting current and open them if it is superior to a defined threshold for a certain amount of time.
```

Comment:
The current limit automaton monitors a line's transiting current and open it if...